packages/interface/src/components/pinned-items/index.js

```diff
@@ -17,7 +16,8 @@ function PinnedItemsSlot( { scope, className, ...props } ) {
 	return (
 		<Slot name={ `PinnedItems/${ scope }` } { ...props }>
 			{ ( fills ) =>
-				! isEmpty( fills ) && (
+				fills &&
+				fills.length && (
```

Comment:
Could use optional chaining here.

Also wanna avoid scenarios where length is `0` and then `0` being printed on the page.

```suggestion
				fills?.length > 0 && (
```

Reply:
Good point, thanks 👍